webapp/channels/src/components/threading/global_threads/thread_item/thread_item.tsx

```diff
@@ -247,11 +273,13 @@ function ThreadItem({
                 </ThreadMenu>
             </div>
             <div
-                aria-readonly='true'
+                ref={previewRef}
                 className='preview'
                 dir='auto'
-                tabIndex={0}
+                tabIndex={hasLink ? 0 : -1}
+                role='button'
```

Comment:
Related to the previous discussion about `handleFormattedTextClick`, does this inner element need to be focusable for links within it to be focusable? Because the click handler is used for inner elements, I wouldn't expect this div to have a role or be focusable even if that'll make the ESLint plugin complain

Reply:
So strictly no, it works. But we do get a warning about adding an `onClick` and `onKeyDown` to an non-interactive element: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-static-element-interactions.md